plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcPageSourceProvider.java

```diff
@@ -37,43 +34,47 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static com.google.common.collect.Iterators.indexOf;
-import static io.trino.plugin.phoenix5.PhoenixClient.MERGE_ROW_ID_COLUMN_NAME;
-import static io.trino.plugin.phoenix5.PhoenixPageSource.ColumnAdaptation;
+import static io.trino.plugin.jdbc.DefaultJdbcMetadata.MERGE_ROW_ID;
 
-public class PhoenixPageSourceProvider
+public class JdbcPageSourceProvider
```

Comment:
Can you please extract refactor to introduce usage of `JdbcPageSourceProvider` as separate commit and PR, then I thin it is simple and we could merge it faster.

Reply:
Filed https://github.com/trinodb/trino/pull/22872